paimon-web-server/src/main/java/org/apache/paimon/web/server/data/model/DatabaseInfo.java

```diff
@@ -32,8 +32,6 @@ public class DatabaseInfo {
 
     private String databaseName;
 
-    private Integer catalogId;
```

Comment:
Why remove this id?

Reply:
> Why remove this id?

catalogId is a useless field here. Paimon's catalog, database, table, etc. do not have id field information. If this change is unnecessary, I will close this PR.